### DISCUSSION TITLE
Email OTP link sometimes working

### DISCUSSION THREAD

**Discussion by JulianSwales** [supabase/supabase#14847] [Questions]
I have a Flutter web app working utilizing the Email OTP solution.. This works most of the time. However, we are getting reports back from some users that they click on the link in the email, they are returned to the login page, vs entering the app.   Most of our users are having no issues, which had me puzzled since it's the same code follow. 
Anyone have any suggestions on what might be the issue. We've been unable to replicate the problem.  
Thanks, Julian

**Comment by polooner** [comment]
Would you be able to share the code? It is probably the way you are handling user sign up/ sign in. Really can't assume or try to help without the code though.
